### PR TITLE
Make some plot series hidden rather than gone

### PR DESCRIPTION
**Pull request by cquiroz**
There was a bit of miss-understanding when doing this task

### CODE REVIEW

File: hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/down.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+alter table "public"."lucumaUserPreferences" rename column "elevationPlotLunarElevationVisible" to "elevationPlotLunarElevationVisiblee";
```
Comment (toddburnside) [inline hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/down.sql:2]: Is the second `e` in `Visiblee` intentional?

**Review by toddburnside — APPROVED** [review]
LGTM

File: hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/down.sql (hunk 1 above)
Comment (cquiroz) [reply hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/down.sql:2]: That was supposed  to be fixed

File: hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/up.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+alter table "public"."lucumaUserPreferences" rename column "elevationPlotParalacticAngleVisible" to "elevationPlotParallacticAngleVisible";
+
+alter table "public"."lucumaUserPreferences" add column "elevationPlotSkyBrightnessVisible" boolean
+ not null default 'true';
+
+alter table "public"."lucumaUserPreferences" add column "elevationPlotLunarElevationVisiblee" boolean
+ not null default 'false';
+
+alter table "public"."lucumaUserPreferences" rename column "elevationPlotLunarElevationVisiblee" to "elevationPlotLunarElevationVisible";
```
Comment (cquiroz) [inline hasura/user-prefs/migrations/default/1712849109532_elevation_plot_series/up.sql:16]: @toddburnside it is fixed here

**Pull request merged**